index.bs

```diff
@@ -4,20 +4,19 @@ Shortname: compression
 Level: none
 Status: w3c/CG-DRAFT
```

Comment:
Is this new in BikeShed? I thought it was just "CG-DRAFT"

Reply:
Apparently the `w3c/` prefix is not needed (citation: https://tabatkins.github.io/bikeshed/#metadata) . Removed.